docs/installation/setup/common_install.rst

```diff
@@ -61,51 +57,18 @@ Since the only user who can connect to a fresh install is the postgres user, her
     sudo -u postgres createuser --superuser $USER
     sudo -u postgres psql
 
-    postgres=# \password $USER
+    postgres=# \password <foo>
 
-Now we can create an ``agdcintegration`` database for testing::
+Now we can create the ``agdcintegration`` and ``odcintegration`` databases for testing::
```

Comment:
Why these names, specifically? Could they be changed to something more descriptive to help the user understand what they're for?

Reply:
They are the databases used for integration testing.  (agdcintegration for tests run against the old default/legacy/postgres index driver, and odcintegration for tests run against the new experimental/postgis index driver).

Reply:
These databases are used for "integration tests", hence the name. But I agree not most obvious name. Ideally this should be captured as a cli tool, something like `datacube bootsrtap --test-db`.

Reply:
I feel people won't necessarily recognise "agdc" as the legacy option and "odc" as the new option. Would something like:
postgresintegration and postgisintegration be more descriptive?

Reply:
It would be fantastic to purge this repo of any remaining references to "AGDC" - that name should no longer appear in any ODC-branded repo IMO

Reply:
I believe all references to 'agdc' stem from the postgres driver which is going to be deprecated in v2 anyway. I'm not sure doing a potentially breaking rename at this stage would be worth it.